lib/BackgroundJob/ExpireGroupVersions.php

```diff
@@ -21,47 +24,20 @@
 
 namespace OCA\GroupFolders\BackgroundJob;
 
-use OCA\GroupFolders\Trash\TrashBackend;
 use OCA\GroupFolders\Versions\GroupVersionsExpireManager;
-use OCA\Files_Trashbin\Expiration;
-use OCP\IConfig;
 
 class ExpireGroupVersions extends \OC\BackgroundJob\TimedJob {
-	const ITEMS_PER_SESSION = 1000;
-
 	/** @var GroupVersionsExpireManager */
 	private $expireManager;
 
-	/** @var TrashBackend */
-	private $trashBackend;
-
-	/** @var Expiration */
-	private $expiration;
-
-	/** @var IConfig */
-	private $config;
-
-	public function __construct(
-		GroupVersionsExpireManager $expireManager,
-		TrashBackend $trashBackend,
-		Expiration $expiration,
-		IConfig $config
-	) {
+	public function __construct(GroupVersionsExpireManager $expireManager) {
 		// Run once per hour
 		$this->setInterval(60 * 60);
 
 		$this->expireManager = $expireManager;
-		$this->trashBackend = $trashBackend;
-		$this->expiration = $expiration;
-		$this->config = $config;
 	}
 
 	protected function run($argument) {
 		$this->expireManager->expireAll();
-		$backgroundJob = $this->config->getAppValue('files_trashbin', 'background_job_expire_trash', 'yes');
-		if ($backgroundJob === 'no') {
-			return;
-		}
-		$this->trashBackend->expire($this->expiration);
```

Comment:
Why is this not needed anymore? Does `expireAll` run `expire` on the trashbin folder?

Reply:
Because this code is in `lib/BackgroundJob/ExpireGroupTrash.php` now